pkg/cmd/cli/cmd/tag.go

```diff
@@ -389,6 +389,9 @@ func (o TagOptions) RunTag() error {
 			default:
 				istag.Tag.From.Name = localRef.NameString()
 				istag.Tag.From.Namespace = o.ref.Namespace
+				if len(o.ref.Namespace) == 0 {
```

Comment:
That could very well be a bug on the server side.

Calling import image in namespace "foo" should not require you to specify
istag.Tag.From.Namespace if namespace == "foo".

On Thu, Aug 18, 2016 at 5:30 PM, Juan Vallejo <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/tag.go
> <https://github.com/openshift/origin/pull/10510#discussion_r75392963>:
>
> > @@ -389,6 +389,9 @@ func (o TagOptions) RunTag() error {
> >  			default:
> >  				istag.Tag.From.Name = localRef.NameString()
> >  				istag.Tag.From.Namespace = o.ref.Namespace
> > +				if len(o.ref.Namespace) == 0 {
>
> @smarterclayton <https://github.com/smarterclayton>
>
> Actually, the from should not have to be set if you are in the current
> namespace.
>
> Hm, it seems to be exiting with the same error described on this PR here
> <https://github.com/openshift/origin/blob/master/pkg/cmd/cli/cmd/tag.go#L419>
> if I do not set a value in istag.Tag.From.Namespace though
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10510/files/ca1f7422ba67657aa0ec0ee598a1705701298932#r75392963>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p6fLQhhxQr25Ny01mT0LUB6Xkyxmks5qhM73gaJpZM4JnnhR>
> .
>